src/components/contact/assets/socialMedia.scss

```diff
@@ -0,0 +1,16 @@
+@import '~theme/base/variables';
+
+.socialMedia {
```

Comment:
The convention within css is to use hypens to split words, ie `social-media`. However, that doesn't work well with css modules. It might be better to go with underscores here. I am not too sure though.

Reply:
do css modules not convert the class name from kebab case to camelCase when imported? always thought that was the case... but i've been in css-in-js land for a long time now so I could be wrong

Reply:
Well, it does end up naming the styles with some random number appended to it to make it unique. So for example, `socialMedia-123abc`. So here we have a mix of camel and kebab. This is not really the issue. It's when you port the styles over to js where things start to look funny. Inside of the components you will see kebab mixed with pascal and snake case. I think it would make sense, if we are going with consistency, to make this snake case since the components are using a mix of pascal/snake to begin with. Although we are getting into what case would be best, also remember that using BEM with css modules doesn't really matter, the styles will always end up being atomic. It is important to follow some kind of naming convention, but it doesn't necessarily have to be how BEM is. But I can see how using BEM can make reading the scss a bit easier when scanning which is nice. Thoughts anyone?

Reply:
@arecvlohe I totally agree with you on use of BEM. With component based architecture, we are not fully benefited with BEM, plus I see a lot of noise in the markup.